promotion/service.go

```diff
@@ -35,10 +35,13 @@ const localEnv = "local"
 var (
 	// toggle for drain retry job
 	enableDrainRetryJob = isDrainRetryJobEnabled()
+	// toggle for grant rewards job
+	enableRewardGrantsJob = isGrantRewardsJobEnabled()
 	// toggle for gemini check status
 	enableGeminiCheckStatus = isRunNextGeminiCheckStatus()
 
 	suggestionTopic       = os.Getenv("ENV") + ".grant.suggestion"
+	rewardsTopic          = os.Getenv("ENV") + ".grant.swaprewards"
```

Comment:
`swapRewardsTopic` might be better here and in similar places because this topic is specific to that.  Swap rewards is a subset of rewards

Reply:
Changed in https://github.com/brave-intl/bat-go/pull/1213/commits/36af29ab409da8c030a17b2330181d5423979a6d